### PR TITLE
add tainted-exec rule in php

### PR DESCRIPTION
**Pull request by hocnc**
Add the tainted-exec rule to detect the command injection vulnerability in PHP

Playground: https://semgrep.dev/playground/s/GG16

### CODE REVIEW

**Review by 0xDC0DE — CHANGES_REQUESTED** [review]
the `.python-version` file should not be part of the PR

**Comment by 0xDC0DE** [comment]
Thanks for your contribution @hocnc ! This looks like a useful rule. It's using taint mode to find user input flowing into a dangerous sink. I think you could update the metadata to a vuln finding rule instead of an audit type rule.

File: php/lang/security/tainted-exec.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    - https://brightsec.com/blog/code-injection-php/
+    - https://www.acunetix.com/websitesecurity/php-security-2/
+    category: security
+    technology:
+    - php
+    owasp:
+    - A03:2021 - Injection
+    cwe2022-top25: true
+    subcategory:
+    - audit
```
Comment (0xDC0DE) [inline php/lang/security/tainted-exec.yaml]:
```suggestion
    - vuln
```

**Comment by 0xDC0DE** [comment]
I still need you to remove the [python/django/security/injection/path-traversal/.python-version](https://github.com/returntocorp/semgrep-rules/pull/3126/files#diff-d1843446b38ce24d87091282f2d586b2512e5e4c6cd2712d24113729636f4b0b) file before I can approve the PR.

**Comment by hocnc** [comment]
i did it

**Review by 0xDC0DE — APPROVED** [review]

**Comment by hocnc** [comment]
Hi @0xDC0DE 

Are there any problems? Why this pull request still is open status ?

Thanks,
Hoc

**Pull request merged**